### DISCUSSION TITLE
[Feature] Rename module | Extend default funcionality

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#13145] [Feature improvement]
To run parallel with trends and improve UX I would like to propose renaming "Internal Tickets" to "Cases" and relate it to all important default modules, so it can track stuff similarly to Opportunities 
but in wider scope.  Opportunity is sales related case and should not cover more complex administrative issues. Ticket can stand alone if covering basic, or specific to related department issue, but should be allowed to be converted to business case if necessary. 
https://www.apm.org.uk/resources/what-is-project-management/what-is-a-business-case/
https://en.wikipedia.org/wiki/Business_case

**Comment by TonyM15** [comment]
Clever. I actually did it myself here. In terms of well designed and fully functional "business operating system" this would be a foundation of all of those who are on the top of the management line. "Business Cases" > mother of modules I would say.

waran70 - you rocks! Cheers man!
